### PR TITLE
Bug 1726639: Consume new viaq plugin to not set level based on stream

### PR DESCRIPTION
**Pull request by jcantrill**
Consume new viaq plugin to resolve https://bugzilla.redhat.com/show_bug.cgi?id=1726639

Note this PR also pulled in updated fluentd

### CODE REVIEW

File: fluentd/Dockerfile [hunk 1]
```diff
@@ -43,7 +43,7 @@ RUN cd ${HOME}/vendored_gem_src/ && ./install-gems.sh && cd / && rm -rf ${HOME}/
 FROM rhel7:7-released
 
 ENV DATA_VERSION=1.6.0 \
-    FLUENTD_VERSION=1.5.1 \
+    FLUENTD_VERSION=1.6.1 \
```
Comment (richm) [inline fluentd/Dockerfile]: 1.6.2

Comment (jcantrill) [reply fluentd/Dockerfile]: fixed

**Pull request merged**